Fatras/include/ActsFatras/Physics/PhotonConversion/PhotonConversion.hpp

```diff
@@ -123,7 +123,7 @@ PhotonConversion::generatePathLimits(generator_t& generator,
 
   // Fast exit if not a photon or the energy is too low
   if (particle.pdg() != Acts::PdgParticle::eGamma ||
-      particle.absoluteMomentum() < m_minimumMomentum)
+      particle.absoluteMomentum() < (2 * kElectronMass))
```

Comment:
why?

Reply:
Photon conversion has a minimal physical threshold, namely the incoming particle has to have enough energy to create two electrons. This just checks that. There is not configurable simulation threshold here, since this is handled at a higher level.